bpdm-bridge-dummy/src/main/resources/application-auth.yml

```diff
@@ -24,6 +24,17 @@ bpdm:
     refresh-url: ${bpdm.security.token-url}
     token-url: ${bpdm.security.auth-server-url}/realms/${bpdm.security.realm}/protocol/openid-connect/token
 
+# Logging Configuration
+logging:
+  pattern: "%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(%5p) %clr([%-15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n%wEx"
```

Comment:
As far as I have tested it, the pattern is not being taking into account. I guess it's because it should be the pattern for the console output. That is: 
logging:
   pattern:
     console:

According to my tests putting the pattern there will lead to it being recognized. I also discovered that the %clr command is not working on this logger. Instead we should use direct color commands. I'm open to any color pattern. Here is a suggestion:

%d{yyyy-MM-dd HH:mm:ss.SSS} %highlight(%-5level) %green([%-10.10X{user:-System}]) %magenta([%-10.10X{request:-No Request}]) %yellow([%-15.15t]) %cyan(%-40.40logger{39}) : %m%n%wEx

Reply:
That color pattern looks good, at least for me. Regarding the Auth properties, should they have something different of the default properties pattern?

Reply:
Pattern is being recognized by Spring Boot now, so that's good. I just noticed the auth profile also has its own pattern. Please remove those changes, the auth profile does not need its own debug behaviour, it can just reuse the default properties